test/org/sagebionetworks/bridge/models/healthdata/HealthDataRecordTest.java

```diff
@@ -387,4 +388,21 @@ public void testSerialization() throws Exception {
         assertFalse(publicJsonMap.containsKey("healthCode"));
         assertEquals("json record ID", publicJsonMap.get("id"));
     }
+
+    @Test
+    public void testTimeZoneFormatter() {
+        testTimeZoneFormatter("+0000", DateTimeZone.UTC);
+        testTimeZoneFormatter("+0000", DateTimeZone.forOffsetHours(0));
+        testTimeZoneFormatter("+0900", DateTimeZone.forOffsetHours(+9));
+        testTimeZoneFormatter("-0800", DateTimeZone.forOffsetHours(-8));
+        testTimeZoneFormatter("+1345", DateTimeZone.forOffsetHoursMinutes(+13, +45));
+        testTimeZoneFormatter("-0330", DateTimeZone.forOffsetHoursMinutes(-3, -30));
```

Comment:
"Z" is valid ISO 8601 time zone, will that work? same with something like "+13:45", will that work? I'm getting this from https://en.wikipedia.org/wiki/ISO_8601#Time_offsets_from_UTC